bundles/org.openhab.binding.onewire/src/test/java/org/openhab/binding/onewire/device/DS2423Test.java

```diff
@@ -54,6 +58,13 @@ public void counterChannelTest() {
         returnValue.add(new DecimalType(1408));
         returnValue.add(new DecimalType(3105));
 
+        final AbstractOwDevice testDevice = this.testDevice;
+        final InOrder inOrder = this.inOrder;
+        if (testDevice==null || inOrder == null) {
+            Assert.fail("prerequisite is null");
```

Comment:
what about using JUnits Assumes? That would make the code here more readable, wouldn't it? The same for the other tests as well.

Reply:
No. Assume just skips the test. If one of these is really null that means something is going severly wrong and the test should fail.

Reply:
If the tests _should_ fail in this case, then I'm ok here :) Thanks for clarifying! :)